app/com/m3/octoparts/repository/ConfigsRepository.scala

```diff
@@ -25,14 +25,34 @@ trait ConfigsRepository {
   def findParamById(id: Long): Future[Option[PartParam]]
 
   /**
-   * Find a single ThreadPoolConfig by id
+   * Find a single ThreadPoolConfig by id.
+   *
+   * All retrieved items are populated with their [[HystrixConfig]]
    */
-  def findThreadPoolConfigById(id: Long): Future[Option[ThreadPoolConfig]]
+  def findThreadPoolConfigById(id: Long): Future[Option[ThreadPoolConfig]] = findAllConfigs().map {
```

Comment:
Hmm, this is fairly heavy because we are doing an uncached (I believe) retrieve of all configs. Maybe there's a better cleaner way to do this via a join or something.